MBProgressHUD.m

```diff
@@ -735,30 +731,19 @@ - (void)updateUIForKeypath:(NSString *)keyPath {
 	[self setNeedsDisplay];
 }
 
-#pragma mark - Notifications
+#pragma mark - Handle orientation changes
 
-- (void)registerForNotifications {
-	NSNotificationCenter *nc = [NSNotificationCenter defaultCenter];
-
-	[nc addObserver:self selector:@selector(statusBarOrientationDidChange:)
-			   name:UIApplicationDidChangeStatusBarOrientationNotification object:nil];
```

Comment:
Shouldn't we just [use](https://developer.apple.com/documentation/uikit/uideviceorientationdidchangenotification) the `UIDeviceOrientationDidChangeNotification` instead?

Reply:
Yes, if that's available that sounds like a much less risky chance to me. Let's do that!